Library/Homebrew/extend/os/mac/extend/ENV/std.rb

```diff
@@ -109,12 +108,8 @@ def macosxsdk(version = nil, formula: nil, testing_formula: false)
       # This is a no-op with macOS SDK 10.15.4 and later.
```

Comment:
Do we still need this?

Reply:
Good question. Worth a try but also not worth blocking PR.

Reply:
I left it as I wasn't sure about our policy on macOS patch releases. I hope no one is still on those versions (10.15.0 - 10.15.3) but decided to play it safe.